snippets/base/templates/base/includes/snippet_as.js

```diff
@@ -5,11 +5,21 @@ var SNIPPET_METRICS_URL = '{{ metrics_url }}';
 var ABOUTHOME_SHOWN_SNIPPET = null;
 var USER_COUNTRY = null;
 var GEO_CACHE_DURATION = 1000 * 60 * 60 * 24 * 30; // 30 days
+var ONBOARDING_TIMEOUT = 2000;
 
-
-(function() {
+(async function() {
     'use strict';
 
+    // See https://github.com/mozilla/activity-stream/blob/master/docs/v2-system-addon/snippets.md
+    // for documentation on the gSnippetsMap API
+
+    // If onboardingFinished is true, there may already be an onboarding snippet
+    // shown so we cannot show a snippet. However, .disableOnboarding() can be
+    // called so that it won't be shown on the *next* new tab
+    if (gSnippetsMap.get("appData.onboardingFinished") !== false) {
```

Comment:
I run `disabledOnboarding()` and it sets `appData.onboardingFinished` but it does not hide the onboarding after opening a new tab. Is this the expected behavior?

(tested on current nighlty)

Reply:
edit: It does hide the onboarding but it does not hide the onboarding icon from the upper left corner. I guess this is the expected behavior in this case.

Reply:
That's correct yes; it only hides the notifications at the bottom